ansible/test.yml

```diff
@@ -114,3 +114,18 @@
       state: directory
   # Setup a port-forward to 8000 and you should be able to login as
   # user:user password:ome
+
+- hosts: scriptsutils
+  roles:
+  - role: scripts-utils
+
+  tasks:
+  - command: python -c 'import reportlab'
+  - command: python -c 'import markdown'
+  - command: python -c 'import os.path; print os.path.exists("/usr/bin/mencoder")'
+    register: value
```

Comment:
This works, but it'd be neater to test for the existence of the file using plain Ansible: Scroll down to

> Should you feel the need to test for existence of files that are not declaratively set by your Ansible configuration, the ‘stat’ module is a great choice:

on http://docs.ansible.com/ansible/test_strategies.html#modules-that-are-useful-for-testing

Alternatively you could just omit all these tests, it's a fairly simple role which installs stuff so if something goes wrong you'd expect to get an error whilst the role was running.